code/modules/projectiles/guns/medbeam.dm

```diff
@@ -111,47 +126,69 @@
 
 /obj/item/gun/medbeam/proc/los_check(atom/movable/user, mob/target)
 	var/turf/user_turf = user.loc
+
+	if(mounted)
+		user_turf = get_turf(user)
+
 	if(!istype(user_turf))
 		return FALSE
+
 	var/obj/dummy = new(user_turf)
-	dummy.pass_flags |= (PASSTABLE|PASSGLASS|PASSGRILLE|PASSFENCE) //Grille/Glass so it can be used through common windows
+	dummy.pass_flags |= (PASSTABLE|PASSGLASS|PASSGRILLE|PASSFENCE) // Grille/Glass so it can be used through common windows
```

Comment:
```suggestion
	dummy.pass_flags |= (PASSTABLE | PASSGLASS | PASSGRILLE | PASSFENCE) // Grille/Glass so it can be used through common windows
```
codestyle